dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/BaseClasses.mo

```diff
@@ -140,7 +135,12 @@ annotation(preferredView = "text",
     U1Pu = ComplexMath.'abs'(terminal1.V);
     s1Pu = Complex(P1Pu, Q1Pu);
     s1Pu = terminal1.V * ComplexMath.conj(terminal1.i);
-    P1Pu = max(min(PMaxPu, P1RefPu), - PMaxPu);
+
+    if (running.value) then
+      P1Pu = max(min(PMaxPu, P1RefPu), - PMaxPu);
+    else
+      P1Pu = 0;
+    end if;
```

Comment:
What we have seen is that you get into trouble when you disconnect the node if you write it this way (because of the s = v * i equation if I remember correctly). 
So even if it is not completely right and not very nice, you have to write:

if running.value then
  P1 = blabla
else
  terminal.i.re = 0
end if;

Besides, it is also consistent with the rest of the library. Remind #1443 .

Reply:
Ok, thanks, done